pdns/test-dns_random_hh.cc

```diff
@@ -2,7 +2,7 @@
 #define BOOST_TEST_NO_MAIN
 
 // Disable this code for gcc 4.8 and lower
-#if (__GNUC__ > 4) || (__GNUC__ == 4 && __GNUC_MINOR__ > 8) || !__GNUC__
+#if (__GNUC__ == 4 && __GNUC_MINOR__ > 8) || !__GNUC__
```

Comment:
This looks wrong, we want to disable this code for versions of GCC lower than 4.8, and this would only enable it for the 4 branch after 4.8.

Reply:
I am not even sure we can build on GCC < 4.8 tbh

Reply:
You are right, will fix it